lib/util.js

```diff
@@ -355,3 +355,48 @@ exports.stripHTML = (html) => {
     .replace(/<.*?>/gi, '')
     .trim();
 };
+
+/**
+ * Match begin and end braces of input JSON, return only json
+ * it works with braced jsons only!
```

Comment:
JSON is always braced.

Reply:
quoted string and unquoted number, and boolean are valid JSON
`JSON.parse('1') // 1`